index.js

```diff
@@ -15,6 +15,7 @@ import { ThrottleGroup } from 'speed-limiter'
 import ConnPool from './lib/conn-pool.js' // browser exclude
 import Torrent from './lib/torrent.js'
 import { NodeServer, BrowserServer } from './lib/server.js'
+import NatAPI from '@silentbot1/nat-api' // browser exclude
```

Comment:
cosmetic - library imports then local imports so this should be above any `somepath/file.js` import